canonical-web-and-design/snapcraft.io
# Update to latest global-nav

## QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004
- Make sure the global nav works as expected
- Yay \o/
